src/components/ScopeTree/ScopeTree.tsx

```diff
@@ -23,24 +23,29 @@ import displayDigit from 'utils/displayDigit'
 import useStyles from './styles'
 import Lock from '@material-ui/icons/Lock'
 import { CheckCircle } from '@material-ui/icons'
+import { practitionerScopeSelector } from 'features/perimeters/perimetersSelector'
 
 type ScopeTreeProps = {
   defaultSelectedItems: ScopeTreeRow[]
   onChangeSelectedItem: (selectedItems: ScopeTreeRow[]) => void
+  restrictToPractitionerPerimeter?: boolean
 }
 
-const ScopeTree: React.FC<ScopeTreeProps> = ({ defaultSelectedItems, onChangeSelectedItem }) => {
+const ScopeTree: React.FC<ScopeTreeProps> = ({
+  defaultSelectedItems,
+  onChangeSelectedItem,
+  restrictToPractitionerPerimeter
+}) => {
   const classes = useStyles()
 
+  const practitionerScopeRows = useAppSelector(practitionerScopeSelector)
   const [openPopulation, onChangeOpenPopulations] = useState<string[]>([])
-  const [rootRows, setRootRows] = useState<ScopeTreeRow[]>([])
-  const [loading, setLoading] = useState(true)
+  const [rootRows, setRootRows] = useState<ScopeTreeRow[]>(practitionerScopeRows)
```

Comment:
Is there a difference between "scopes" and "perimeters"?
What is `restrictToPractitionerPerimeter` for if we already `setRootRows` with data specific to the practitioner?

Reply:
The way I see it, it's pretty much the same

`restrictToPractitionerPerimeter` is needed to load all api organizations if it is falsy. This ScopeTree component is used both in creation cohort feature (where it needs to fetch all orgas) and in practitioner's perimeter flow (where only the practitioner's allowed organizations are displayed)